corehq/apps/cleanup/management/commands/populate_sql_model_from_couch_model.py

```diff
@@ -238,3 +259,18 @@ def _migrate_doc(self, doc):
             model, created = self.update_or_create_sql_object(doc)
             action = "Creating" if created else "Updated"
             logger.info("{} model for doc with id {}".format(action, doc["_id"]))
+
+    def _get_couch_doc_count_for_domain(self, domain):
```

Comment:
Nitpick: it'd be nice to have this expect a list so it's more identical to the sql version.

Reply:
df5ee1060b8